src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/appsettings.json

```diff
@@ -1,11 +1,10 @@
-﻿{
+{
     "OTEL_EXPORTER_OTLP_ENDPOINT": "REPLACE_ME",
     "APPLICATIONINSIGHTS_CONNECTION_STRING": "InstrumentationKey=REPLACE_ME",
 	"ServiceBusConfiguration": {
 		"Sender": {
 			"Topics": [
 				{
-					"Name": "sbt-menu-events",
```

Comment:
is this change needed?

Reply:
Appologies, no its not, that was accidentally included from a Stash.  Will revert.